Ms2Database/DBClasses/Ms2DBContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Ms2Database.DBClasses
+{
+    public class Ms2DBContext : DbContext
+    {
+        public Ms2DBContext() : base("name=DatabaseConnectionString")
+        {
+            //Database.SetInitializer<Ms2DBContext>(new DropCreateDatabaseAlways<Ms2DBContext>());
+        }
+        public DbSet<Account> Accounts { get; set; }
```

Comment:
Can this live above the constructor, if possible?

Reply:
Will give it a try when I get home.

Reply:
just tried during work using Atom editor. Its giving a syntax error when i move it above the constructor. So it will have to stay below the constructor.